docs/deploy.md

```diff
@@ -176,12 +176,23 @@ to the cluster.
     (hip)$ kubectl get node
     ```
 
-## Useful Commands
+### Get access to the AWS Console:
+
+If you need access to the AWS Console, create a bookmark with this URL:
+
+https://signin.aws.amazon.com/switchrole?roleName=CaktusAccountAccessRole-Admins&account=061553509755&displayName=Philly-PDPH
+
+If you are logged into the AWS console with your `caktus` account, then that URL will
+switch you to the PDPH subaccount.
+
+Reference: https://github.com/caktus/caktus-hosting-services/blob/main/docs/aws-assumerole.md#aws-accounts
+
+### Useful Commands
 
 To see configured ansible variable like ``k8s_environment_variables`` you can use the following invoke command.
 
 ```sh
-(hip)$ inv staging info.get-ansible-vars
+(hip)$ inv staging info.print-ansible-vars
```

Comment:
Note: running this command complains about not having a region set up (`You must specify a region. You can also configure your region by running "aws configure".`) Adding the region for philly-hip in `~/.aws/credentials` fixed the issue.

Reply:
Ahh, I have mine set in a separate file in `~/.aws/config` for some reason. I'll update the docs.